src/rhsmlib/services/register.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, cp):
 
     def register(self, org, activation_keys=None, environment=None, force=None, name=None, consumerid=None,
             **kwargs):
```

Comment:
Okay, I just added a check to error out on a Register call if the consumer is already registered.  Good catch.